ClientCore/Settings/UserINISettings.cs

```diff
@@ -143,8 +139,8 @@ protected UserINISettings(IniFile iniFile)
         public BoolSetting WindowedMode { get; private set; }
         public BoolSetting BorderlessWindowedMode { get; private set; }
         public BoolSetting BackBufferInVRAM { get; private set; }
-        public IntSetting ClientResolutionX { get; private set; }
-        public IntSetting ClientResolutionY { get; private set; }
+        public IntSetting ClientResolutionX { get; set; }
+        public IntSetting ClientResolutionY { get; set; }
```

Comment:
The reading from INI code was moved, not removed. The part that IS removed is the grabbing of the default values using the WinForms method Screen.PrimaryScreen which is not cross-platform.
This code was moved to Startup.Execute() since that's the earliest we can grab those values from XNA instead using GraphicsAdapter.DefaultAdapter.